### PR TITLE
TDP-3738 add new SendDtmf fields

### PR DESCRIPTION
**Pull request by rmello-daitan**
* New supported fields to SendDtmf verb

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3738-new-SendDtmf-fields.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/sendDtmf.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ The SendDtmf verb is used to play DTMF digits in the call.
 ### Attributes
 | ATTRIBUTE | Description |
 |:----------|:------------|
-| None      | None        |
+| toneDuration | (optional) The length (in milliseconds) of each DTMF tone. Default value is 200. Range: decimal values between 50 - 5000. |
+| toneInterval | (optional) The duration of silence (in milliseconds) following each DTMF tone. Default value is 400. Range: decimal values between 50 - 5000. |
```
Comment (ghost) [inline voice/bxml/verbs/sendDtmf.md:12]: I think it's worth mentioning here that "wW," all replace the default tone duration.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3738-new-SendDtmf-fields.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3738-new-SendDtmf-fields.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3738-new-SendDtmf-fields.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3738-new-SendDtmf-fields.s3-website-us-east-1.amazonaws.com/

**Review by ghost — APPROVED** [review]

**Pull request merged**